.gitmodules

```diff
@@ -1,3 +1,4 @@
 [submodule "cgranges"]
 	path = cgranges
 	url = https://github.com/lh3/cgranges
+	ignore = dirty
```

Comment:
Every time this submodule is compiled (`poetry install`) untracked files are added to the directory.

This tells Git to ignore those untracked compiled files.